implementations.md

```diff
@@ -11,11 +11,8 @@ If you know of any associated projects that are not listed here, please file a p
 
 * [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
 
-## Bundle authoring
+## Testing & Tools
 
 * [kunalkushwaha/octool](https://github.com/kunalkushwaha/octool) - A config linter and validator.
-* [mrunalp/ocitools](https://github.com/mrunalp/ocitools) - A config generator.
-
-## Testing
-
+* [opencontainers/ocitools](https://github.com/opencontainers/ocitools) - A config generator and runtime testing framework.
```

Comment:
Agree, 'testing&tools' is a better description here.  

I think @mrunalp agreed ocitools should cover three areas currently.(I cannot find the related conversation in issue/PR)
1. generate
2. bundle validate
3. runtime validate